app/assets/stylesheets/_sponsor_application.scss

```diff
@@ -0,0 +1,80 @@
+*,
+*:after,
+*:before {
+  box-sizing: border-box;
+}
+
+.steps-container {
+  overflow: hidden;
```

Comment:
Properties should be ordered border-left, border-right, counter-reset, margin, overflow, padding, white-space, width